grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -456,8 +456,24 @@ class Order implements Serializable {
         return !hasInvoice && isPrepaymentRequired && hasActiveItemsOrAdjustments
     }
 
+    /*
```

Comment:
I think you need javadocs to start with `/**` (two asterisks) to have them work correctly in IDEs and other tools. Good job with the documentation here though!